src/app/pagamentoplanos/component/pagamentoplanos.component.ts

```diff
@@ -7,9 +9,119 @@ import { Component, OnInit } from '@angular/core';
 })
 export class PagamentoplanosComponent implements OnInit {
 
-  constructor() { }
+  constructor(
+    private scriptService: ExternalService,
+    private pagamentoSerice: PagamentoplanosService
+  ) { }
 
   ngOnInit(): void {
+    this.scriptService.load('MercadoPago').then(data => {
+  }).catch(error => console.log(error));
+    //@ts-ignore
+    const mp = new MercadoPago('YOUR_PUBLIC_KEY');
+    const cardForm = mp.cardForm({
+      amount: "100.5",
+      autoMount: true,
+      form: {
+        id: "form-checkout",
+        cardholderName: {
+          id: "form-checkout__cardholderName",
+          placeholder: "Titular do cartão",
+        },
+        cardholderEmail: {
+          id: "form-checkout__cardholderEmail",
+          placeholder: "E-mail",
+        },
+        cardNumber: {
+          id: "form-checkout__cardNumber",
+          placeholder: "Número do cartão",
+        },
+        cardExpirationMonth: {
+          id: "form-checkout__cardExpirationMonth",
+          placeholder: "Mês de vencimento",
+        },
+        cardExpirationYear: {
+          id: "form-checkout__cardExpirationYear",
+          placeholder: "Ano de vencimento",
+        },
+        securityCode: {
+          id: "form-checkout__securityCode",
+          placeholder: "Código de segurança",
+        },
+        installments: {
+          id: "form-checkout__installments",
+          placeholder: "Parcelas",
+        },
+        identificationType: {
+          id: "form-checkout__identificationType",
+          placeholder: "Tipo de documento",
+        },
+        identificationNumber: {
+          id: "form-checkout__identificationNumber",
+          placeholder: "Número do documento",
+        },
+        issuer: {
+          id: "form-checkout__issuer",
+          placeholder: "Banco emissor",
+        },
+      },
+      callbacks: {
+        onFormMounted: (error: any) => {
+          if (error) return console.warn("Form Mounted handling error: ", error);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
          if (error) {return console.warn("Form Mounted handling error: ", error);}
```